test/helpers/story/drivers/dapp-story-test-driver.ts

```diff
@@ -66,14 +71,26 @@ export default class DappStoryTestDriver extends StoryTestDriver {
     const actionType = action.actionType
     switch (actionType) {
       case 'AAVE': {
-        const newTest = new Test('AAVE Lend DAPP', (async () => {
-          expect(1).to.equal(1)
-        }))
+        const newTest = new Test('AAVE Lend DAPP', async () => {
+          const { getNamedSigner } = hre
+          const borrower = await getNamedSigner('borrower')
+          const loan = await LoanStoryTestDriver.getLoan(hre, borrower)
+          let shouldPass = true
+          if (!loan) {
+            shouldPass = false
+          }
+          if (shouldPass) {
+            // await LoanStoryTestDriver.takeOutLoan(hre, {})
+            await DappStoryTestDriver.lendAave(hre, loan)
+          } else {
+            await DappStoryTestDriver.lendAave(hre, loan)
+          }
```

Comment:
why are these the same for both scenarios?

Reply:
fixed